src/harkerrobolib/subsystems/LEDController.java

```diff
@@ -0,0 +1,103 @@
+package harkerrobolib.subsystems;
+
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import java.awt.Color;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+/** 
+ * Instantiates methods for basic functionalities of an led strip
+ * 
+ * @author Ada Praun-Petrovic
+ * @since April 12, 2020
+ */
+public class LEDController {
+
+    private final AddressableLED mLED;
```

Comment:
Minor thing, don't start instance variable names with `m`, since we don't follow it in other places in the codebase